### ISSUE TITLE
Templates create compile errors in solutions with Central Package Management (CPM)

### ISSUE THREAD

**Issue by cremor** [dotnet/sdk#32367]

**Comment by mkArtakMSFT** [comment]
@marcpopMSFT this seems to be a tooling ask, specifically for adding a project reference to an existing project. Can you please direct this to the right owners? Thanks!

**Comment by marcpopMSFT** [comment]
CC @JonDouglas @baronfel 
this seems like a reasonable ask of the dotnet add package command but this specific issue is about templates. This would be a feature request on the template engine for templates to support CPM and have dotnet detect CPM and pass a default value through to the template that controlled the PackageReferences included based on that value.

**Comment by baronfel** [comment]
I agree it's a template-engine area concern - we have some larger-scale questions about how Packages are managed in templates. Most folks have hardcoded PackageReferences, so either the _entire_ package ecosystem needs to update templates to conditionally-set versions, or we need to do something more first-class in the engine.

**Comment by baronfel** [comment]
If these were item templates (i.e. a project already existed) then we could use a `bind` symbol to check if the project supported CPM and then change the template content based on that, but instead we probably need to actually do work here.

**Comment by cremor** [comment]
So should this issue be transferred to https://github.com/dotnet/templating? If yes, could someone please do that?
